File: java-code/src-java/inc/brody/ship/Battleship.java
package inc.brody.ship;

import com.badlogic.gdx.Game;

public class Battleship extends Game {
	private Game game;

	@Override
	public void create () {
		Assets.load();
		setScreen(new GameScreen(this));
	}

	@Override
	public void dispose() {
		Assets.dispose();
	}
}

File: core/src/inc/brody/bs/Battleship.kt
package inc.brody.bs

import com.badlogic.gdx.Game
import inc.brody.bs.objects.Assets
import inc.brody.bs.screens.GameScreen

class Battleship : Game() {

    override fun create() {
        Assets.load()
        setScreen(GameScreen(this))
    }

    override fun dispose() {
        Assets.dispose()
    }

}
